cob_sick_s300/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ project(cob_sick_s300)
 find_package(catkin REQUIRED COMPONENTS diagnostic_msgs roscpp sensor_msgs std_msgs)
 
```

Comment:
and another [boost](https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_sick_s300/ros/src/cob_sick_s300.cpp#L74)?